src/features/content-explorer/content-explorer/ContentExplorerFolderTreeBreadcrumbs.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, FormattedMessage } from 'react-intl';
+
+import Arrow16 from '../../../icon/fill/Arrow16';
+import Button from '../../../components/button';
+import DropdownMenu from '../../../components/dropdown-menu/DropdownMenu';
+import IconFolderTree from '../../../icons/general/IconFolderTree';
+import { Menu, MenuItem } from '../../../components/menu';
+
+import { FoldersPathPropType } from '../prop-types';
+import messages from '../messages';
+import {
+    FOLDER_TREE_ICON_HEIGHT,
+    FOLDER_TREE_ICON_WIDTH,
+    BREADCRUMB_ARROW_ICON_HEIGHT,
+    BREADCRUMB_ARROW_ICON_WIDTH,
+    BREADCRUMB_ARROW_ICON_VIEWBOX,
+} from './constants';
+
+const ContentExplorerFolderTreeBreadcrumbs = ({
+    foldersPath,
+    intl: { formatMessage, formatNumber },
+    isFolderTreeButtonDisabled = false,
+    numTotalItems,
+    onBreadcrumbClick,
+}) => {
+    const currentFolderName = foldersPath[foldersPath.length - 1].name;
+    return (
+        <div className="bdl-ContentExplorerFolderTreeBreadcrumbs">
+            <DropdownMenu>
+                <Button
+                    aria-label={formatMessage(messages.clickToViewPath)}
+                    className="bdl-ContentExplorerFolderTreeBreadcrumbs-button"
+                    isDisabled={isFolderTreeButtonDisabled}
+                    title={formatMessage(messages.filePath)}
+                    type="button"
+                >
+                    <IconFolderTree height={FOLDER_TREE_ICON_HEIGHT} width={FOLDER_TREE_ICON_WIDTH} />
+                </Button>
+                <Menu>
+                    {foldersPath.map((folder, i) => (
```

Comment:
oops didn't realize this till now but don't we want to exclude the last item from the dropdown?

Reply:
Hmm that is an interesting idea. I created this under the impression that it would be shown because the old breadcrumb design also showed the current folder that you are in and nothing would happen if you clicked it. But now that it is a dropdown, it could be worth removing

Reply:
you can double check with the designer if the dropdown should include it or not. i imagine clicking the last item in dropdown might not do anything because the user is already in the folder

Reply:
We have decided to exclude the last item from the dropdown, and we will not render the folder tree icon until we are past the Search Results page, since there is nothing to click back to when viewing the Search Results